src/controller/http-api-router.js

```diff
@@ -23,44 +23,44 @@ class HttpApiRouter {
 
     async initializeListeners() {
         // POST REQUESTS
-        this.httpClientModule.post(HTTP_API_ROUTES.PUBLISH, (req, res) => {
+        this.httpClientModuleInterface.post(HTTP_API_ROUTES.PUBLISH, (req, res) => {
             this.publishController.handleHttpApiPublishRequest(req, res);
         });
```

Comment:
Is there a particular reason for wrapping route names inside of HTTP_API_ROUTES constant?

In my opinion this causes router obfuscation, since we can't see route name directly in the file and therefore can't search file by route name.